docs/organizing-and-exploring-results-in-the-ui/organizing-experiments.rst

```diff
@@ -28,37 +27,182 @@ You can create tag(s), which you assign to experiments. Later, you can quickly f
+Use "`manage columns`" button to decide what to display:
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/manage-columns.png
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/manage-columns.png
+   :alt: Manage columns
 
 .. note::
 
-    Neptune automatically proposes columns based on what is different between experiments. This helps you see what changed quickly.
+    Learn more how to log different types of meta-data: :ref:`What objects can you log to Neptune <what-you-can-log>`.
+
+Auto-proposed columns
+^^^^^^^^^^^^^^^^^^^^^
+Note, that neptune automatically proposes columns based on what is different between experiments. This helps you see what changed quickly. Suggested columns are the right-most columns in the dashboard. See example below:
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/suggested-columns.png
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/suggested-columns.png
+   :alt: Suggested columns
+
+Sort dashboard by column
+^^^^^^^^^^^^^^^^^^^^^^^^
+You can decide over which column to sort the dashboard. Use arrows in the column header to do it:
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/sort-columns.png
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/sort-columns.png
+   :alt: Sort columns
+
+Decide how to display column data
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+For each column individually, you can decide how its data is displayed. Click on the cog icon |cog| and select display format:
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/column-display-format.png
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/column-display-format.png
+   :alt: column display format
+
+Reorder columns and adjust their width
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Moreover, you can reorder columns and adjust their width.
+
+.. image:: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/col-order-width.gif
+   :target: ../_static/images/organizing-and-exploring-results-in-the-ui/experiment-dashboard/col-order-width.gif
+   :alt: columns order and width
 
 Grouping experiments
 --------------------
-
 You can group experiments by one or more column(s). The dashboard displays the selected columns, allowing you to make in-group and across-groups analysis of the experiments. Each group is represented by the first experiment that appears according to the sorting order. After opening it, each group shows at most 10 experiments - all experiments can be viewed by clicking **Show all**.
```

Comment:
I'd add space after some sentences here -> it is a big text blob right now.

Reply:
ok